apps/aecore/src/aec_conductor.erl

```diff
@@ -206,6 +212,61 @@ get_block_by_hash(Hash) when is_binary(Hash) ->
 get_block_by_height(Height) ->
     gen_server:call(?SERVER, {get_block_by_height, Height}).
 
+-spec get_block_pair_by_height(integer(), integer()) -> {ok, #block{}, #block{}} | {error, atom()}.
+get_block_pair_by_height(Height1, Height2) ->
+    gen_server:call(?SERVER, {get_block_pair, height, Height1, Height2}).
+
+-spec get_block_pair_by_hash(binary(), binary()) -> {ok, #block{}, #block{}} | {error, atom()}.
+get_block_pair_by_hash(Hash1, Hash2) ->
+    gen_server:call(?SERVER, {get_block_pair, hash, Hash1, Hash2}).
+
+max_block_range() -> ?MAXIMUM_BLOCK_RANGE.
+
+get_block_range_by_height(Height1, Height2) ->
+    get_block_range(Height1, Height2, fun get_block_pair_by_height/2).
+
+get_block_range_by_hash(Hash1, Hash2) ->
+    get_block_range(Hash1, Hash2, fun get_block_pair_by_hash/2).
+
+get_block_range(HFrom, HTo, AtomicGetBlockPairFun) when is_function(AtomicGetBlockPairFun, 2) ->
```

Comment:
Get the block pairs and send the result as an argument to the function rather than apply a fun.